README_LINUX.md

```diff
@@ -1,211 +1,199 @@
-Welcome to SuperCollider for Linux!
-===================================
+Building SuperCollider on Linux
+===============================
 
-Intro
------
+Build requirements
+------------------
 
-SuperCollider is a synthesis engine (scsynth or supernova) and programming language (sclang), originally Mac-based but now very widely used and actively developed on Linux.
+All of these are required for building scsynth and supernova.
+
+- [gcc][gcc] >= 4.8
+- [cmake][cmake] >= 3.5: Cross-platform build system.
+- [libsndfile][libsndfile] >= 1.0: Soundfile I/O.
+- [libjack][libjack]: Development headers for the JACK Audio Connection Kit.
+- [fftw][fftw] >= 3.0: FFT library.
+- [libxt][libxt]: X toolkit intrinsics, required for UGens such as `MouseX`. (Support for building the servers without X is in progress.)
+- [libavahi-client][libavahi-client]: For zero-configuration networking. Can be turned off by setting the CMake flag `NO_AVAHI=ON`, but turning it off is not recommended.
+
+
+[gcc]: http://www.gnu.org/software/gcc
+[libjack]: http://www.jackaudio.org/
+[cmake]: http://www.cmake.org
+[libsndfile]: http://www.mega-nerd.com/libsndfile
+[fftw]: http://www.fftw.org
+[libavahi-client]: http://www.avahi.org
+[libxt]: http://www.X.org
+
+Recommended packages
+--------------------
+
+For sclang and scide:
+
+- [Qt][Qt] >= 5.7: Cross-platform GUI library, required for the IDE and for sclang's Qt GUI kit. It's best to get the latest Qt 5.x version.
+- QtWebEngine: required for the IDE.
+- [git][git]: Required for sclang's Quarks system.
+- [ALSA][ALSA]: Linux sound library, required for sclang MIDI support.
+- [libudev][libudev]: Device manager library, required for HID support.
+- [Linux kernel][Linux kernel] >= 2.6: Required for LID support.
+- [libreadline][libreadline] >= 5: Required for sclang's CLI interface.
+
+[Qt]: http://qt-project.org
+[ALSA]: http://www.alsa-project.org
+[libudev]: http://www.freedesktop.org/software/systemd/man/libudev.html
+[libreadline]: http://savannah.gnu.org/projects/readline
+[Linux kernel]: http://www.kernel.org
+[git]: https://git-scm.com/
```

Comment:
More duplication?

Reply:
i don't think there's any duplication here -- this is just the "footnote style" for entering markdown links

Reply:
:man_facepalming:

Reply:
@snappizz this could as well have been links to not bloat the README ;)